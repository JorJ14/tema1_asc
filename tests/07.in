{
    "products": {
        "id1": {
            "product_type": "Coffee",
            "name": "Indonezia",
            "acidity": 5.05,
            "roast_level": "MEDIUM",
            "price": 1
        },
        "id2": {
            "product_type": "Coffee",
            "name": "Brasil",
            "acidity": 5.09,
            "roast_level": "MEDIUM",
            "price": 7
        },
        "id3": {
            "product_type": "Coffee",
            "name": "Ethiopia",
            "acidity": 5.09,
            "roast_level": "MEDIUM",
            "price": 10
        },
        "id4": {
            "product_type": "Tea",
            "name": "Wild Cherry",
            "type": "Black",
            "price": 4
        },
        "id5": {
            "product_type": "Tea",
            "name": "Cactus fig",
            "type": "Green",
            "price": 3
        }
    },
    "producers": [
        {
            "name": "prod1",
            "products": [
                [
                    "id2",
                    2,
                    0.4
                ],
                [
                    "id1",
                    3,
                    0.39
                ],
                [
                    "id3",
                    3,
                    0.37
                ]
            ],
            "republish_wait_time": 0.16
        },
        {
            "name": "prod2",
            "products": [
                [
                    "id4",
                    2,
                    0.25
                ]
            ],
            "republish_wait_time": 0.17
        },
        {
            "name": "prod3",
            "products": [
                [
                    "id5",
                    2,
                    0.2
                ],
                [
                    "id2",
                    3,
                    0.14
                ],
                [
                    "id3",
                    3,
                    0.17
                ]
            ],
            "republish_wait_time": 0.08
        },
        {
            "name": "prod4",
            "products": [
                [
                    "id1",
                    3,
                    0.16
                ],
                [
                    "id4",
                    1,
                    0.12
                ]
            ],
            "republish_wait_time": 0.21
        },
        {
            "name": "prod5",
            "products": [
                [
                    "id3",
                    3,
                    0.4
                ],
                [
                    "id5",
                    3,
                    0.4
                ]
            ],
            "republish_wait_time": 0.30
        }
    ],
    "consumers": [
        {
            "name": "cons1",
            "retry_wait_time": 0.46,
            "carts": [
                [
                    {
                        "type": "add",
                        "product": "id5",
                        "quantity": 2
                    },
                    {
                        "type": "add",
                        "product": "id3",
                        "quantity": 5
                    },
                    {
                        "type": "add",
                        "product": "id4",
                        "quantity": 5
                    },
                    {
                        "type": "remove",
                        "product": "id4",
                        "quantity": 1
                    }
                ],
                [
                    {
                        "type": "add",
                        "product": "id3",
                        "quantity": 2
                    },
                    {
                        "type": "add",
                        "product": "id2",
                        "quantity": 2
                    },
                    {
                        "type": "add",
                        "product": "id4",
                        "quantity": 2
                    }
                ],
                [
                    {
                        "type": "add",
                        "product": "id4",
                        "quantity": 2
                    },
                    {
                        "type": "add",
                        "product": "id1",
                        "quantity": 2
                    },
                    {
                        "type": "add",
                        "product": "id5",
                        "quantity": 1
                    }
                ]
            ]
        },
        {
            "name": "cons2",
            "retry_wait_time": 0.36,
            "carts": [
                [
                    {
                        "type": "add",
                        "product": "id5",
                        "quantity": 2
                    },
                    {
                        "type": "add",
                        "product": "id3",
                        "quantity": 2
                    },
                    {
                        "type": "add",
                        "product": "id4",
                        "quantity": 5
                    },
                    {
                        "type": "remove",
                        "product": "id3",
                        "quantity": 2
                    }
                ],
                [
                    {
                        "type": "add",
                        "product": "id3",
                        "quantity": 3
                    },
                    {
                        "type": "add",
                        "product": "id1",
                        "quantity": 5
                    }
                ],
                [
                    {
                        "type": "add",
                        "product": "id3",
                        "quantity": 2
                    },
                    {
                        "type": "add",
                        "product": "id2",
                        "quantity": 1
                    },
                    {
                        "type": "remove",
                        "product": "id2",
                        "quantity": 1
                    }
                ],
                [
                    {
                        "type": "add",
                        "product": "id3",
                        "quantity": 1
                    },
                    {
                        "type": "add",
                        "product": "id4",
                        "quantity": 1
                    }
                ],
                [
                    {
                        "type": "add",
                        "product": "id1",
                        "quantity": 1
                    },
                    {
                        "type": "add",
                        "product": "id5",
                        "quantity": 2
                    },
                    {
                        "type": "add",
                        "product": "id4",
                        "quantity": 5
                    }
                ]
            ]
        },
        {
            "name": "cons3",
            "retry_wait_time": 0.19,
            "carts": [
                [
                    {
                        "type": "add",
                        "product": "id5",
                        "quantity": 1
                    }
                ],
                [
                    {
                        "type": "add",
                        "product": "id1",
                        "quantity": 4
                    }
                ],
                [
                    {
                        "type": "add",
                        "product": "id1",
                        "quantity": 1
                    },
                    {
                        "type": "add",
                        "product": "id4",
                        "quantity": 3
                    },
                    {
                        "type": "add",
                        "product": "id3",
                        "quantity": 1
                    }
                ]
            ]
        },
        {
            "name": "cons4",
            "retry_wait_time": 0.08,
            "carts": [
                [
                    {
                        "type": "add",
                        "product": "id2",
                        "quantity": 5
                    },
                    {
                        "type": "add",
                        "product": "id1",
                        "quantity": 4
                    }
                ],
                [
                    {
                        "type": "add",
                        "product": "id4",
                        "quantity": 3
                    },
                    {
                        "type": "add",
                        "product": "id2",
                        "quantity": 4
                    },
                    {
                        "type": "add",
                        "product": "id5",
                        "quantity": 5
                    }
                ],
                [
                    {
                        "type": "add",
                        "product": "id1",
                        "quantity": 3
                    },
                    {
                        "type": "add",
                        "product": "id2",
                        "quantity": 5
                    },
                    {
                        "type": "add",
                        "product": "id5",
                        "quantity": 3
                    }
                ]
            ]
        },
        {
            "name": "cons5",
            "retry_wait_time": 0.26,
            "carts": [
                [
                    {
                        "type": "add",
                        "product": "id1",
                        "quantity": 4
                    },
                    {
                        "type": "add",
                        "product": "id4",
                        "quantity": 5
                    },
                    {
                        "type": "add",
                        "product": "id3",
                        "quantity": 5
                    },
                    {
                        "type": "remove",
                        "product": "id4",
                        "quantity": 4
                    }
                ],
                [
                    {
                        "type": "add",
                        "product": "id2",
                        "quantity": 1
                    },
                    {
                        "type": "add",
                        "product": "id1",
                        "quantity": 3
                    },
                    {
                        "type": "add",
                        "product": "id5",
                        "quantity": 1
                    },
                    {
                        "type": "remove",
                        "product": "id2",
                        "quantity": 1
                    }
                ],
                [
                    {
                        "type": "add",
                        "product": "id5",
                        "quantity": 3
                    },
                    {
                        "type": "add",
                        "product": "id3",
                        "quantity": 5
                    }
                ],
                [
                    {
                        "type": "add",
                        "product": "id4",
                        "quantity": 1
                    },
                    {
                        "type": "add",
                        "product": "id5",
                        "quantity": 1
                    },
                    {
                        "type": "add",
                        "product": "id3",
                        "quantity": 5
                    }
                ]
            ]
        },
        {
            "name": "cons6",
            "retry_wait_time": 0.29,
            "carts": [
                [
                    {
                        "type": "add",
                        "product": "id4",
                        "quantity": 4
                    },
                    {
                        "type": "remove",
                        "product": "id4",
                        "quantity": 4
                    }
                ],
                [
                    {
                        "type": "add",
                        "product": "id1",
                        "quantity": 1
                    },
                    {
                        "type": "add",
                        "product": "id2",
                        "quantity": 3
                    },
                    {
                        "type": "add",
                        "product": "id4",
                        "quantity": 2
                    },
                    {
                        "type": "remove",
                        "product": "id4",
                        "quantity": 1
                    }
                ]
            ]
        },
        {
            "name": "cons7",
            "retry_wait_time": 0.36,
            "carts": [
                [
                    {
                        "type": "add",
                        "product": "id4",
                        "quantity": 4
                    },
                    {
                        "type": "add",
                        "product": "id1",
                        "quantity": 2
                    }
                ],
                [
                    {
                        "type": "add",
                        "product": "id1",
                        "quantity": 4
                    },
                    {
                        "type": "add",
                        "product": "id4",
                        "quantity": 3
                    },
                    {
                        "type": "add",
                        "product": "id3",
                        "quantity": 1
                    }
                ],
                [
                    {
                        "type": "add",
                        "product": "id1",
                        "quantity": 5
                    }
                ],
                [
                    {
                        "type": "add",
                        "product": "id5",
                        "quantity": 2
                    },
                    {
                        "type": "remove",
                        "product": "id5",
                        "quantity": 1
                    }
                ]
            ]
        },
        {
            "name": "cons8",
            "retry_wait_time": 0.09,
            "carts": [
                [
                    {
                        "type": "add",
                        "product": "id1",
                        "quantity": 1
                    },
                    {
                        "type": "add",
                        "product": "id3",
                        "quantity": 3
                    },
                    {
                        "type": "add",
                        "product": "id2",
                        "quantity": 2
                    },
                    {
                        "type": "remove",
                        "product": "id2",
                        "quantity": 1
                    }
                ],
                [
                    {
                        "type": "add",
                        "product": "id1",
                        "quantity": 2
                    },
                    {
                        "type": "add",
                        "product": "id5",
                        "quantity": 3
                    },
                    {
                        "type": "remove",
                        "product": "id1",
                        "quantity": 1
                    }
                ],
                [
                    {
                        "type": "add",
                        "product": "id4",
                        "quantity": 5
                    },
                    {
                        "type": "add",
                        "product": "id5",
                        "quantity": 2
                    },
                    {
                        "type": "add",
                        "product": "id2",
                        "quantity": 2
                    },
                    {
                        "type": "remove",
                        "product": "id5",
                        "quantity": 1
                    }
                ],
                [
                    {
                        "type": "add",
                        "product": "id3",
                        "quantity": 3
                    },
                    {
                        "type": "remove",
                        "product": "id3",
                        "quantity": 3
                    }
                ]
            ]
        },
        {
            "name": "cons9",
            "retry_wait_time": 0.08,
            "carts": [
                [
                    {
                        "type": "add",
                        "product": "id3",
                        "quantity": 2
                    }
                ],
                [
                    {
                        "type": "add",
                        "product": "id3",
                        "quantity": 2
                    },
                    {
                        "type": "add",
                        "product": "id4",
                        "quantity": 3
                    },
                    {
                        "type": "add",
                        "product": "id5",
                        "quantity": 1
                    },
                    {
                        "type": "remove",
                        "product": "id3",
                        "quantity": 1
                    }
                ],
                [
                    {
                        "type": "add",
                        "product": "id5",
                        "quantity": 3
                    },
                    {
                        "type": "add",
                        "product": "id1",
                        "quantity": 4
                    },
                    {
                        "type": "remove",
                        "product": "id5",
                        "quantity": 2
                    }
                ],
                [
                    {
                        "type": "add",
                        "product": "id5",
                        "quantity": 4
                    },
                    {
                        "type": "remove",
                        "product": "id5",
                        "quantity": 3
                    }
                ],
                [
                    {
                        "type": "add",
                        "product": "id1",
                        "quantity": 4
                    },
                    {
                        "type": "remove",
                        "product": "id1",
                        "quantity": 3
                    }
                ]
            ]
        },
        {
            "name": "cons10",
            "retry_wait_time": 0.17,
            "carts": [
                [
                    {
                        "type": "add",
                        "product": "id5",
                        "quantity": 4
                    }
                ],
                [
                    {
                        "type": "add",
                        "product": "id2",
                        "quantity": 2
                    }
                ]
            ]
        }
    ],
    "marketplace": {
        "queue_size_per_producer": 30
    }
}
