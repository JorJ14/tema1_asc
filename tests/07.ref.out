cons1 bought Coffee(name='Brasil', price=7, acidity=5.09, roast_level='MEDIUM')
cons1 bought Coffee(name='Brasil', price=7, acidity=5.09, roast_level='MEDIUM')
cons1 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons1 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons1 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons1 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons1 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons1 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons1 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons1 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons1 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons1 bought Tea(name='Cactus fig', price=3, type='Green')
cons1 bought Tea(name='Cactus fig', price=3, type='Green')
cons1 bought Tea(name='Cactus fig', price=3, type='Green')
cons1 bought Tea(name='Wild Cherry', price=4, type='Black')
cons1 bought Tea(name='Wild Cherry', price=4, type='Black')
cons1 bought Tea(name='Wild Cherry', price=4, type='Black')
cons1 bought Tea(name='Wild Cherry', price=4, type='Black')
cons1 bought Tea(name='Wild Cherry', price=4, type='Black')
cons1 bought Tea(name='Wild Cherry', price=4, type='Black')
cons1 bought Tea(name='Wild Cherry', price=4, type='Black')
cons1 bought Tea(name='Wild Cherry', price=4, type='Black')
cons10 bought Coffee(name='Brasil', price=7, acidity=5.09, roast_level='MEDIUM')
cons10 bought Coffee(name='Brasil', price=7, acidity=5.09, roast_level='MEDIUM')
cons10 bought Tea(name='Cactus fig', price=3, type='Green')
cons10 bought Tea(name='Cactus fig', price=3, type='Green')
cons10 bought Tea(name='Cactus fig', price=3, type='Green')
cons10 bought Tea(name='Cactus fig', price=3, type='Green')
cons2 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons2 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons2 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons2 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons2 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons2 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons2 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons2 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons2 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons2 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons2 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons2 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons2 bought Tea(name='Cactus fig', price=3, type='Green')
cons2 bought Tea(name='Cactus fig', price=3, type='Green')
cons2 bought Tea(name='Cactus fig', price=3, type='Green')
cons2 bought Tea(name='Cactus fig', price=3, type='Green')
cons2 bought Tea(name='Wild Cherry', price=4, type='Black')
cons2 bought Tea(name='Wild Cherry', price=4, type='Black')
cons2 bought Tea(name='Wild Cherry', price=4, type='Black')
cons2 bought Tea(name='Wild Cherry', price=4, type='Black')
cons2 bought Tea(name='Wild Cherry', price=4, type='Black')
cons2 bought Tea(name='Wild Cherry', price=4, type='Black')
cons2 bought Tea(name='Wild Cherry', price=4, type='Black')
cons2 bought Tea(name='Wild Cherry', price=4, type='Black')
cons2 bought Tea(name='Wild Cherry', price=4, type='Black')
cons2 bought Tea(name='Wild Cherry', price=4, type='Black')
cons2 bought Tea(name='Wild Cherry', price=4, type='Black')
cons3 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons3 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons3 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons3 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons3 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons3 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons3 bought Tea(name='Cactus fig', price=3, type='Green')
cons3 bought Tea(name='Wild Cherry', price=4, type='Black')
cons3 bought Tea(name='Wild Cherry', price=4, type='Black')
cons3 bought Tea(name='Wild Cherry', price=4, type='Black')
cons4 bought Coffee(name='Brasil', price=7, acidity=5.09, roast_level='MEDIUM')
cons4 bought Coffee(name='Brasil', price=7, acidity=5.09, roast_level='MEDIUM')
cons4 bought Coffee(name='Brasil', price=7, acidity=5.09, roast_level='MEDIUM')
cons4 bought Coffee(name='Brasil', price=7, acidity=5.09, roast_level='MEDIUM')
cons4 bought Coffee(name='Brasil', price=7, acidity=5.09, roast_level='MEDIUM')
cons4 bought Coffee(name='Brasil', price=7, acidity=5.09, roast_level='MEDIUM')
cons4 bought Coffee(name='Brasil', price=7, acidity=5.09, roast_level='MEDIUM')
cons4 bought Coffee(name='Brasil', price=7, acidity=5.09, roast_level='MEDIUM')
cons4 bought Coffee(name='Brasil', price=7, acidity=5.09, roast_level='MEDIUM')
cons4 bought Coffee(name='Brasil', price=7, acidity=5.09, roast_level='MEDIUM')
cons4 bought Coffee(name='Brasil', price=7, acidity=5.09, roast_level='MEDIUM')
cons4 bought Coffee(name='Brasil', price=7, acidity=5.09, roast_level='MEDIUM')
cons4 bought Coffee(name='Brasil', price=7, acidity=5.09, roast_level='MEDIUM')
cons4 bought Coffee(name='Brasil', price=7, acidity=5.09, roast_level='MEDIUM')
cons4 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons4 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons4 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons4 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons4 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons4 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons4 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons4 bought Tea(name='Cactus fig', price=3, type='Green')
cons4 bought Tea(name='Cactus fig', price=3, type='Green')
cons4 bought Tea(name='Cactus fig', price=3, type='Green')
cons4 bought Tea(name='Cactus fig', price=3, type='Green')
cons4 bought Tea(name='Cactus fig', price=3, type='Green')
cons4 bought Tea(name='Cactus fig', price=3, type='Green')
cons4 bought Tea(name='Cactus fig', price=3, type='Green')
cons4 bought Tea(name='Cactus fig', price=3, type='Green')
cons4 bought Tea(name='Wild Cherry', price=4, type='Black')
cons4 bought Tea(name='Wild Cherry', price=4, type='Black')
cons4 bought Tea(name='Wild Cherry', price=4, type='Black')
cons5 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons5 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons5 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons5 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons5 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons5 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons5 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons5 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons5 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons5 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons5 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons5 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons5 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons5 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons5 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons5 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons5 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons5 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons5 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons5 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons5 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons5 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons5 bought Tea(name='Cactus fig', price=3, type='Green')
cons5 bought Tea(name='Cactus fig', price=3, type='Green')
cons5 bought Tea(name='Cactus fig', price=3, type='Green')
cons5 bought Tea(name='Cactus fig', price=3, type='Green')
cons5 bought Tea(name='Cactus fig', price=3, type='Green')
cons5 bought Tea(name='Wild Cherry', price=4, type='Black')
cons5 bought Tea(name='Wild Cherry', price=4, type='Black')
cons6 bought Coffee(name='Brasil', price=7, acidity=5.09, roast_level='MEDIUM')
cons6 bought Coffee(name='Brasil', price=7, acidity=5.09, roast_level='MEDIUM')
cons6 bought Coffee(name='Brasil', price=7, acidity=5.09, roast_level='MEDIUM')
cons6 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons6 bought Tea(name='Wild Cherry', price=4, type='Black')
cons7 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons7 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons7 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons7 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons7 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons7 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons7 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons7 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons7 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons7 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons7 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons7 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons7 bought Tea(name='Cactus fig', price=3, type='Green')
cons7 bought Tea(name='Wild Cherry', price=4, type='Black')
cons7 bought Tea(name='Wild Cherry', price=4, type='Black')
cons7 bought Tea(name='Wild Cherry', price=4, type='Black')
cons7 bought Tea(name='Wild Cherry', price=4, type='Black')
cons7 bought Tea(name='Wild Cherry', price=4, type='Black')
cons7 bought Tea(name='Wild Cherry', price=4, type='Black')
cons7 bought Tea(name='Wild Cherry', price=4, type='Black')
cons8 bought Coffee(name='Brasil', price=7, acidity=5.09, roast_level='MEDIUM')
cons8 bought Coffee(name='Brasil', price=7, acidity=5.09, roast_level='MEDIUM')
cons8 bought Coffee(name='Brasil', price=7, acidity=5.09, roast_level='MEDIUM')
cons8 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons8 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons8 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons8 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons8 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons8 bought Tea(name='Cactus fig', price=3, type='Green')
cons8 bought Tea(name='Cactus fig', price=3, type='Green')
cons8 bought Tea(name='Cactus fig', price=3, type='Green')
cons8 bought Tea(name='Cactus fig', price=3, type='Green')
cons8 bought Tea(name='Wild Cherry', price=4, type='Black')
cons8 bought Tea(name='Wild Cherry', price=4, type='Black')
cons8 bought Tea(name='Wild Cherry', price=4, type='Black')
cons8 bought Tea(name='Wild Cherry', price=4, type='Black')
cons8 bought Tea(name='Wild Cherry', price=4, type='Black')
cons9 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons9 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons9 bought Coffee(name='Ethiopia', price=10, acidity=5.09, roast_level='MEDIUM')
cons9 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons9 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons9 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons9 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons9 bought Coffee(name='Indonezia', price=1, acidity=5.05, roast_level='MEDIUM')
cons9 bought Tea(name='Cactus fig', price=3, type='Green')
cons9 bought Tea(name='Cactus fig', price=3, type='Green')
cons9 bought Tea(name='Cactus fig', price=3, type='Green')
cons9 bought Tea(name='Wild Cherry', price=4, type='Black')
cons9 bought Tea(name='Wild Cherry', price=4, type='Black')
cons9 bought Tea(name='Wild Cherry', price=4, type='Black')
